front/app/modules/commercial/smart_groups/components/UserFilterConditions/rules.ts

```diff
@@ -76,6 +76,8 @@ type TFollowPredicate =
   | 'is_not_folder'
   | 'is_not_initiative'
   | 'is_not_idea'
+  | 'is_not_topic'
+  | 'is_not_area'
```

Comment:
Improving the type safety highlighted that these were missing for this type, so I added them.